### ISSUE TITLE
Inline ignore

### ISSUE THREAD

**Issue by kytwb** [eslint/eslint#1926]
Would it be possible to add inline ignore so instead of:

```javascript
/*eslint-disable no-unused-vars*/
var should = require('should')
/*eslint-enable no-unused-vars*/
```

We could have:

```javascript
var should = require('should') // eslint-ignore no-unused-vars
```

**Comment by nzakas** [comment]
We actually have a proposal almost complete: https://github.com/eslint/eslint/pull/1840

**Comment by kytwb** [comment]
Awesome, didn't thought checking the existing PR.
I let you close or keep open until the PR merge, as you prefer.

**Comment by nzakas** [comment]
Implemented in #1840.
